package.json

```diff
@@ -1,22 +1,21 @@
 {
   "dependencies": {
+    "@scalar-labs/scalardl-javascript-sdk-base": "^1.0.0",
     "google-protobuf": "^3.6.1",
-    "grpc-web": "^1.0.4",
-    "jsrsasign": "^8.0.12"
+    "grpc-web": "^1.0.4"
   },
   "name": "@scalar-labs/scalardl-web-client-sdk",
-  "version": "0.0.1",
+  "version": "1.0.1",
```

Comment:
Sorry, others are 1.1.0, so maybe 1.1.0 ?

Reply:
Yes. It should be 1.1.0 !